akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -250,6 +250,11 @@ object Source {
    * The actor will be stopped when the stream is completed, failed or canceled from downstream,
    * i.e. you can watch it to get notified when that happens.
    *
+   * An async boundary is automatically created after the returned actorRef source, so downstream
+   * operations will not occur in the same thread as the Actor ingesting messages. If you select too
```

Comment:
I respect your position, and disagree. Users should, and do care. This is a major point of surprise that should be documented clearly. When I'm using something, the first question I ask (and, in my experience, others ask) is: how can I prevent this from dropping messages? It's not unreasonable for people to assume that it's safe to set bufferSize to a very small value because they think that the downstream should never block (because there's a buffer, or conflate, or something).

It was a major point of surprise for me, and I've been using Akka streams since pre 1.0. I don't think it would be wise to ignore this point of feedback.